src/main/resources/hudson/tasks/Mailer/help-authentication_zh_CN.html

```diff
@@ -1,3 +1,3 @@
-<div>
-  当发送邮件时使用SMTP认证.如果你的环境需要使用SMTP认证,在这里指定其用户名和密码.
+<div>
+  当发送邮件时使用SMTP认证.如果你的环境需要使用SMTP认证,在这里指定其用户名和密码.
```

Comment:
Do I dare ask?

Reply:
Not sure. local git tool only reports file being moved, no internal changes.